.gitignore

```diff
@@ -106,3 +106,5 @@ venv.bak/
 .idea
```

Comment:
This stuff should go in your [global gitignore](https://stackoverflow.com/q/7335420/4759433).